modules/installing-rhv-example-install-config-yaml.adoc

```diff
@@ -70,8 +70,27 @@ platform:
 publish: External
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed12345 AAAA...
+compute:
+  platform:
+    ovirt:
+      ..
+      sparse: false # add this line
+      format: raw   # add this line
+controlPlane:
+  platform:
+    ovirt:
+      ..
+      sparse: false # add this line <1>
+      format: raw   # add this line <2>
 ----
 
+<1> Sparse - Setting this option to `false` enables preallocation of disks. Default: `true`. Setting `sparse` to `true` with `format` set to `raw` is not available for block storage domains. The 'raw' format writes the entire virtual disk to the underlying physical disk.
+<2> Format - Can be set to `cow` or `raw`. Default: `cow`. The 'cow' format is optimized for virtual machines.
++
+[NOTE]
+====
+Preallocating disks on file storage domains will write zeroes to the file. This may not actually preallocate disks depending on the underlying storage.
```

Comment:
The ISG recommends using "may" only to refer to permission. You could replace "may" with "might".

Reply:
Does this note relate specifically to the last call out? If so, you could add some context in the note to make that clear.